GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerFixture/GVFSUpgradeReminderTests.cs

```diff
@@ -62,6 +63,51 @@ public void NoReminderForLeftOverDownloads()
             this.VerifyUpgradeVerbStopsReminder();
         }
 
+        [TestCase]
+        public void UpgradeTimerScheduledOnServiceStart()
+        {
+            this.RestartService();
+            Thread.Sleep(TimeSpan.FromSeconds(15));

```

Comment:
Rather than always waiting 15 seconds, could we do something like check for the message every 1 second a max of 15 times?  That way we might be able to get through this test faster.

Reply:
Yes, that is a better approach. Incorporated it.